src/index.ts

```diff
@@ -74,6 +74,7 @@ class ElectronGoogleOAuth2 extends EventEmitter {
     if (!scopes.includes('email')) scopes.push('email');
     this.scopes = scopes;
     this.options = { ...defaultElectronGoogleOAuth2Options, ...options };
+    this.prompt = [];
```

Comment:
Fixes a :bug: introduced in 79515429b94927f8e5e13e5fd6e002d289bce7c5. `prompt` needs to be initialized.